apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -453,3 +445,11 @@ type ResearchOutputParsedTags = {
   environments: string[];
   subtype?: string;
 };
+
+export type ResearchOutputFetchOptions = {
+  take?: number;
+  skip?: number;
+  search?: string;
+  filter?: ResearchOutputFilter;
```

Comment:
since we already have `includeDrafts` isn't your task simply to make sure ONLY drafts are coming through? I guess you could expand this filter:
https://github.com/yldio/asap-hub/blob/23c7ae9481a5a043f564226be09b1e11da4591ef/apps/crn-server/src/controllers/research-outputs.ts#L427-L434
and add a status option there (after documentType, title, link)